src/coreclr/ToolBox/superpmi/superpmi/jitinstance.cpp

```diff
@@ -346,6 +346,12 @@ JitInstance::Result JitInstance::CompileMethod(MethodContext* MethodToCompile, i
                     matchesTargetArch = (targetArch == SPMI_TARGET_ARCHITECTURE_ARM64);
                     break;
 
+#ifdef TARGET_UNIX
```

Comment:
Instead of adding this ifdef, it would be better to define `IMAGE_FILE_MACHINE_LOONGARCH64` on Windows.

Reply:
I'm sorry, where is the defination on windows?

Reply:
I meant that you can define `IMAGE_FILE_MACHINE_LOONGARCH64` in a place that shared between Windows and non-Windows so that these ifdefs are not necessary.

src/coreclr/inc/clrnt.h may be a good place to add `IMAGE_FILE_MACHINE_LOONGARCH64` definition that is shared between Windows and non-Windows.

Try adding this:
```
#ifndef IMAGE_FILE_MACHINE_LOONGARCH64
#define IMAGE_FILE_MACHINE_LOONGARCH64 0x6264
#endif
```
to https://github.com/dotnet/runtime/blob/main/src/coreclr/inc/clrnt.h#L1032

Reply:
> #ifndef IMAGE_FILE_MACHINE_LOONGARCH64
> #define IMAGE_FILE_MACHINE_LOONGARCH64 0x6264
> #endif

Thanks～

Reply:
https://github.com/dotnet/runtime/pull/62886#discussion_r773577691

But the compiling error is still exist.

Reply:
> src/coreclr/inc/clrnt.h may be a good place to add `IMAGE_FILE_MACHINE_LOONGARCH64` definition that is shared between Windows and non-Windows.

It seems that it doesn't work.